benchexec/tablegenerator/test_integration/expected/smt.diff.statistics.tex

```diff
@@ -0,0 +1,67 @@
+% The following definition defines a command for each value.
+% The command name is the concatenation of the first six arguments.
+% To override this definition, define \StoreBenchExecResult with \newcommand before including this file.
+% Arguments: benchmark name, run-set name, category, status, column name, statistic, value
+\providecommand\StoreBenchExecResult[7]{\expandafter\newcommand\csname#1#2#3#4#5#6\endcsname{#7}}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Score}{}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Total}{}{Sum}{3}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Correct}{}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Correct}{True}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Correct}{False}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Correct}{Unconfirmed}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{CorrectUnconfirmed}{True}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{CorrectUnconfirmed}{False}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Wrong}{}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Wrong}{True}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Status}{Wrong}{False}{Sum}{0}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Sum}{0.155786}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Min}{0.009091}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Max}{0.125825}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Avg}{0.05192866666666666666666666667}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Median}{0.02087}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Stdev}{0.05247340450382671923076363801}%
+\StoreBenchExecResult{Mathsat}{}{Cputime}{Total}{}{Unit}{s}%
```

Comment:
Right, it matches what we have in the [HTML table](https://sosy-lab.github.io/benchexec/example-table/svcomp-simple-cbmc-cpachecker.table.html#/). Status is `0` because there are 0 results of the respective category, but other statistics are shown as `-` in order to show that this is the sum over the empty set instead of just a sum that happens to be 0.

For the LaTeX output, keeping the explicit status values with count 0 might be useful for some users that want to confirm that they did not overlook anything (they could add the equivalent to "count for category wrong is 0" to their LaTeX code, for example), so we'll keep it.